atlassian/jira.py

```diff
@@ -973,7 +973,10 @@ def assign_issue(self, issue, account_id=None):
         """
         base_url = self.resource_url("issue")
         url = "{base_url}/{issue}/assignee".format(base_url=base_url, issue=issue)
-        data = {"accountId": account_id}
+        data = {"name": account_id}
+        log.warning(
```

Comment:
This warning should be removed.

Reply:
Sure @Spacetown , but before could you help with the reason please ?

Just following the similar convention used in the other functions within same `jira` module like,

1. https://github.com/atlassian-api/atlassian-python-api/blob/master/atlassian/jira.py#L958

2. https://github.com/atlassian-api/atlassian-python-api/blob/master/atlassian/jira.py#L1031

and many more.

warning is lower logger level than info, so shouldn't be noise as per my knowledge.

If we have a conflict, would like @gonchik to suggest a way forward.

Reply:
Please consider https://docs.python.org/3/howto/logging.html:
- warning: `Issue a warning regarding a particular runtime event`
- info: `Report events that occur during normal operation of a program (e.g. for status monitoring or fault investigation)`

In normal operation no messages are good messages.

Reply:
Ok thanks for the reply,  I agree to what your referring to but your missing the point I am making. 😄 

I didn't get your reply to my query for existing warning logs placed. Could you help out for that please ?

Is `issue_add_watcher` a normal operation? - https://github.com/atlassian-api/atlassian-python-api/blame/master/atlassian/jira.py#L958
You could explain this line especially since you added it.

I will happily clear this but then does it make sense to have the existing warning logs also removed and from which functions I am unclear on? seems a select few? please help on those too with an Issue then.

@gonchik could you help out here please if this conversation is taking too long 😆

Reply:
@isaac-philip warning is more noise than info. Warnings and errors are visible per default, Infos are only activated in verbose mode..
I'm with you that the other places should also be to changed.
If the function is called the user is warned that the desired action is done but this is the intention.